packages/expo-test-runner/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  root: true,
+  extends: ['universe/node'],
+  overrides: [
+    {
+      files: ['**/__tests__/*'],
+    },
+  ],
+  globals: {
+    jasmine: false,
+  },
+  rules: {
+    'no-constant-condition': ['warn', { checkLoops: false }],
+    'sort-imports': [
+      'warn',
+      {
+        ignoreCase: true,
+        ignoreDeclarationSort: true,
+      },
+    ],
+    'flowtype/no-types-missing-file-annotation': 'off',
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+};
```

Comment:
It can be replaced by
```
module.exports = require('expo-module-scripts/eslintrc.base.js');
```
when you add `expo-module-scripts` as a `devDependency`.